<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Federico  Agostini</title>
<meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->
<link rel="shortcut icon" href="/assets/img/favicon.ico">
<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/">

<!-- JQuery -->
<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>


<!-- Theming-->

  <script src="/assets/js/theme.js"></script>
  <!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>






    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/">
       <span class="font-weight-bold">Federico</span>   Agostini
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              About
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/teaching/">
                Appointments
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/projects/">
                Projects
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/publications/">
                Publications
                
              </a>
          </li>
          
          
          
          
          
          
          
          
          
            <div class = "toggle-container">
              <a id = "light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
              </a>
            </div>
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">
     <span class="font-weight-bold">Federico</span>   Agostini
    </h1>
     <p class="desc"><p><a href="https://www.scilifelab.se">Science for Life Laboratory</a>, <a href="https://ki.se/en/mbb/about-the-department-of-medical-biochemistry-and-biophysics">Department of Medical Biochemistry and Biophysics</a>,  <a href="https://www.ki.se">Karolinska Institutet</a></p> <p>Stockholm, Sweden</p>
</p>
  </header>

  <article>
    
    <div class="profile float-left">
      
        <img class="img-fluid z-depth-1 rounded" src="/assets/img/prof_pic.jpg">
      
      
        <div class="address">
          <p>Tomtebodavägen 23a</p> <p>171 65 Solna, Sweden</p>

        </div>
      
    </div>
    

    <div class="clearfix">
      <p>I studied biotechnologies in Italy, obtained my PhD in Barcelona and did my first postdoc in London. During the years, I’ve been working with RNA binding proteins and transcription, but now I’m more focused on understanding how the genome is organised inside the nucleus. To do this, I analyse different sources of data and try to make a sense out of them.</p>

    </div>

    

    
      <div class="publications">
  <h2>selected publications</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NAT COMMUN</abbr>
    
  
  </div>

  <div id="bonetti2020radicl" class="col-sm-8">
    
      <div class="title">RADICL-seq identifies general and cell type–specific principles of genome-wide RNA-chromatin interactions</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Bonetti, Alessandro,
                
              
            
          
        
          
          
          
          
          
          
            
              
                <em>Agostini, Federico</em>,
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Suzuki, Ana Maria,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Hashimoto, Kosuke,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Pascarella, Giovanni,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Gimenez, Juliette,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Roos, Leonie,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Nash, Alex J,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Ghilotti, Marco,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Cameron, Christopher J F,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Valentine, Matthew,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Medvedeva, Yulia A.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Noguchi, Shuhei,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Agirre, Eneritz,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Kashi, Kaori,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Samudyata, ,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Luginbühl, Joachim,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Cazzoli, Riccardo,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Agrawal, Saumya,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Luscombe, Nicholas M.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Blanchette, Mathieu,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Kasukawa, Takeya,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Hoon, Michiel,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Arner, Erik,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Lenhard, Boris,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Plessy, Charles,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Castelo-Branco, Gonçalo,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Orlando, Valerio,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Carninci, Piero
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Nature Communications</em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Mammalian genomes encode tens of thousands of noncoding RNAs. Most noncoding transcripts exhibit nuclear localization and several have been shown to play a role in the regulation of gene expression and chromatin remodeling. To investigate the function of such RNAs, methods to massively map the genomic interacting sites of multiple transcripts have been developed; however, these methods have some limitations. Here, we introduce RNA And DNA Interacting Complexes Ligated and sequenced (RADICL-seq), a technology that maps genome-wide RNA–chromatin interactions in intact nuclei. RADICL-seq is a proximity ligation-based methodology that reduces the bias for nascent transcription, while increasing genomic coverage and unique mapping rate efficiency compared with existing methods. RADICL-seq identifies distinct patterns of genome occupancy for different classes of transcripts as well as cell type–specific RNA-chromatin interactions, and highlights the role of transcription in the establishment of chromatin structure.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BIORXIV</abbr>
    
  
  </div>

  <div id="agostini2020intergenic" class="col-sm-8">
    
      <div class="title">Intergenic RNA mainly derives from nascent transcripts of known genes</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                <em>Agostini, Federico</em>,
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Zagalak, Julian A,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Attig, Jan,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Ule, Jernej,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Luscombe, Nicholas M,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Federico, Agostini,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Julian, Zagalak,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Jan, Attig,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Jernej, Ule,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and M, Luscombe Nicholas
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>bioRxiv</em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Background: Eukaryotic genomes undergo pervasive transcription, leading to the production of many types of stable and unstable RNAs. Transcription is not restricted to regions with annotated gene features but includes almost any genomic context. Currently, the source and function of most RNAs originating from intergenic regions in the human genome remains unclear. Results: We hypothesised that many intergenic RNA can be ascribed to the presence of as-yet unannotated genes or the &#039;fuzzy&#039; transcription of known genes that extends beyond the annotated boundaries. To elucidate the contributions of these two sources, we assembled a dataset of &gt;2.5 billion publicly available RNA-seq reads across 5 human cell lines and multiple cellular compartments to annotate transcriptional units in the human genome. About 80% of transcripts from unannotated intergenic regions can be attributed to the fuzzy transcription of existing genes; the remaining transcripts originate mainly from putative long non-coding RNA loci that are rarely spliced. We validated the transcriptional activity of these intergenic RNA using independent measurements, including transcriptional start sites, chromatin signatures, and genomic occupancies of RNA polymerase II in various phosphorylation states. We also analysed the nuclear localisation and sensitivities of intergenic transcripts to nucleases to illustrate that they tend to be rapidly degraded either &#039;on-chromatin&#039; by XRN2 or &#039;off-chromatin&#039; by the exosome. Conclusions: We provide a curated atlas of intergenic RNAs that distinguishes between alternative processing of well annotated genes from independent transcriptional units based on the combined analysis of chromatin signatures, nuclear RNA localisation and degradation pathways.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NAT BIOTECHNOL</abbr>
    
  
  </div>

  <div id="girelli2020gpseq" class="col-sm-8">
    
      <div class="title">GPSeq reveals the radial organization of chromatin in the cell nucleus</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Girelli, Gabriele,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Custodio, Joaquin,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Kallas, Tomasz,
                
              
            
          
        
          
          
          
          
          
          
            
              
                <em>Agostini, Federico</em>,
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Wernersson, Erik,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Spanjaard, Bastiaan,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Mota, Ana,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Kolbeinsdottir, Solrun,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Gelali, Eleni,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Crosetto, Nicola,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Bienko, Magda
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Nature Biotechnology</em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>With the exception of lamina-associated domains, the radial organization of chromatin in mammalian cells remains largely unexplored. Here we describe genomic loci positioning by sequencing (GPSeq), a genome-wide method for inferring distances to the nuclear lamina all along the nuclear radius. GPSeq relies on gradual restriction digestion of chromatin from the nuclear lamina toward the nucleus center, followed by sequencing of the generated cut sites. Using GPSeq, we mapped the radial organization of the human genome at 100-kb resolution, which revealed radial patterns of genomic and epigenomic features and gene expression, as well as A and B subcompartments. By combining radial information with chromosome contact frequencies measured by Hi-C, we substantially improved the accuracy of whole-genome structure modeling. Finally, we charted the radial topography of DNA double-strand breaks, germline variants and cancer mutations and found that they have distinctive radial arrangements in A and B subcompartments. We conclude that GPSeq can reveal fundamental aspects of genome architecture.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">CELL</abbr>
    
  
  </div>

  <div id="attig2018heteromeric" class="col-sm-8">
    
      <div class="title">Heteromeric RNP Assembly at LINEs Controls Lineage-Specific RNA Processing</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Attig, Jan,
                
              
            
          
        
          
          
          
          
          
          
            
              
                <em>Agostini, Federico</em>,
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Gooding, Clare,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Chakrabarti, Anob M,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Singh, Aarti,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Haberman, Nejc,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Zagalak, Julian A,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Emmett, Warren,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Smith, Christopher W.J.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Luscombe, Nicholas M,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Ule, Jernej
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Cell</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Long mammalian introns make it challenging for the RNA processing machinery to identify exons accurately. We find that LINE-derived sequences (LINEs) contribute to this selection by recruiting dozens of RNA-binding proteins (RBPs) to introns. This includes MATR3, which promotes binding of PTBP1 to multivalent binding sites within LINEs. Both RBPs repress splicing and 3′ end processing within and around LINEs. Notably, repressive RBPs preferentially bind to evolutionarily young LINEs, which are located far from exons. These RBPs insulate the LINEs and the surrounding intronic regions from RNA processing. Upon evolutionary divergence, changes in RNA motifs within LINEs lead to gradual loss of their insulation. Hence, older LINEs are located closer to exons, are a common source of tissue-specific exons, and increasingly bind to RBPs that enhance RNA processing. Thus, LINEs are hubs for the assembly of repressive RBPs and also contribute to the evolution of new, lineage-specific transcripts in mammals.</p>
    </div>
    
  </div>
</div>
</li></ol>
</div>

    

    
    <div class="social">
      <span class="contact-icon text-center">
  <a href="mailto:%66%65%64%65%72%69%63%6F.%61%67%6F%73%74%69%6E%69@%6B%69.%73%65"><i class="fas fa-envelope"></i></a>
  <a href="https://orcid.org/0000-0002-5453-2737" target="_blank" title="ORCID"><i class="ai ai-orcid"></i></a>
  <a href="https://scholar.google.com/citations?user=https://scholar.google.com/citations?user=ZZmNuLEAAAAJ&hl=en" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
  
  
  <a href="https://github.com/fagostini" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
  <a href="https://www.linkedin.com/in/federico-agostini-46bbb7146" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
  
  
  
  
  
  
  
  
</span>

      <div class="contact-note"></div>
    </div>
    
  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2021 Federico  Agostini.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank">Unsplash</a>.

    
    
  </div>
</footer>



  </body>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
